libminifi/src/core/PropertyValidation.cpp

```diff
@@ -27,6 +27,7 @@ std::shared_ptr<PropertyValidator> StandardValidators::VALID = std::make_shared<
 StandardValidators::StandardValidators() {
   INVALID = std::make_shared<AlwaysValid>(false, "INVALID");
   INTEGER_VALIDATOR = std::make_shared<IntegerValidator>("INTEGER_VALIDATOR");
+  UNSIGNED_INT_VALIDATOR = std::make_shared<UnsignedIntValidator>("UNSIGNED_INT_VALIDATOR");
```

Comment:
To ensure compatibility with MiNiFi java via C2 (having matching manifest in case of validators) please make sure about UInt32 validator exists in NiFi and it has the same name. ( I can help with this)

Reply:
https://github.com/apache/nifi/blob/master/nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java#L292 :sweat:

Reply:
Fair, but not sure whether that's 32 or 64 bits.

Reply:
Java `int` is 32 bit signed, but the validator filters all the negatives so it's effectively 31 bit.

Reply:
Side note: IIRC, Java 8 provides some Unsinged methods for Integer class so that it can work in range 0..2^32-1.